.github/workflows/test-setup-sigstore-env.yml

```diff
@@ -0,0 +1,36 @@
+name: test setup-sigstore-env
+on:
+  workflow_call:
```

Comment:
Can we pair this down run when relevant files are updated?

Reply:
fixed